### ISSUE TITLE
Home office desk for engineering 2020

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#444]

**Comment by 130s** [comment]
Requirement:
- Horizontally (y-axis) not constrained (wide). Not sure but x-axis depth might not be super important.
   - Monitors don't need to sit on the desk. Desk is where physical work happens, e.g. writing, placing, moving objects (physical work doesn't happen in/on monitors).
- No legs interrupting the move of my legs.
- Desktop surface is uniformely the same height. No height diff. E.g. if the solution is L-shape desk, height of the top for main surface and "return" surface are the same.
- (Option) Cool looking.

All these make me go down to the route for L-shape desk. After extensive research I picked https://www.facebook.com/marketplace/item/492553184748885 (one of the items the seller posted, which later turned out [IKEA Galant](https://www.businessinsider.com/ikea-is-killing-the-galant-desk-2014-7) that is coordinated as L-shape).

Bought it. The seller told me while I was on the way that it is missing some screws. Sellers were kind and helpful, saying those screws can be easily found at home shop. Went to Home Depot nearby and both 2 persons at the fasterner section said one of the screws I brought in is very specific and not common at all. The other type of screw is also not exactly the same length (but found one likely with same width).

Turned out it's most likely this table https://www.ikea.com/assembly_instructions/galant-table__2_Q112_PUB.PDF. Missing screws:
- IKEA 111034
   - https://www.ebay.com/p/710823656
- IKEA 100227
   - https://picclick.com/IKEA-METRIC-SCREW-METRIC-M8-HEX6-HEAD-CAP-192745002980.html
   - https://www.ebay.com/itm/1x-IKEA-METRIC-SCREW-M8-HEX6-HEAD-CAP-MC6S-40mm-L-PART-100227-/192745002980

**Comment by 130s** [comment]
I tried the screws the Home Depot clerks suggested as "alternatives" (Home Depot's ID are 564604: M8-1.25 x 60mm, 564800: M10-1.5 x 35mm), and both 2 types kind of work but not exactly (e.g. wobbly). So I ended up ordering from ebay.com the exact models. Will take about 10 days to receive them.

Total monetary cost that is taking just for the missing screws:
- $10 at Home Depot for alternatives that later turned out not sufficient.
- $48 at ebay.com incl. shipping.

Lessong: When buying 2nd hand furniture, don't buy if screws are missing. Better to have originals if possible.
